glean_parser/schemas/metrics.2-0-0.schema.yaml

```diff
@@ -435,7 +435,7 @@ definitions:
         title: Unit
         description: |
           The unit of the metric, for metrics that don't already require a
-          meaningful unit, such as `time_unit`.
+          meaningful unit, such as `quantity`.
```

Comment:
`time_unit` is the meaningful unit. But I agree this wording could be updated.

Reply:
Woops. accidental change.
But yes, re-reading that it's hard to understand.